source/agora/test/NetworkManager.d

```diff
@@ -180,6 +180,7 @@ unittest
 
     // node test will accept its blocks from node_validator,
     // as the blocks in node_bad do not pass validation
-    retryFor(node_test.getBlockHeight() == 10, 4.seconds);
+    retryFor(node_test.getBlockHeight() == 10, 4.seconds,
+        node_test.getBlockHeight().to!string);
```

Comment:
Yo

Reply:
Sorry. But it was useful during testing. :P